### ISSUE TITLE
Use ids instead of classes for single use styles

### ISSUE THREAD

**Issue by cjessett** [brett099/prj-rev-bwfs-dasmoto#3]
(Subtlety) Instead of creating a class for a single use of this style attribute, you should use the `id` attribute. In general, css classes are reused on multiple elements throughout the page.
https://github.com/brett099/prj-rev-bwfs-dasmoto/blob/4a6ef4e37544354c12e2175b42b73563d5d18958/arts%26crafts/resources/css/style.css#L15

**Comment by brett099** [comment]
Yeah i should have switched those to ids. I didnt because i remembered the specificity lesson on css and i figured id use less specific classes
